src/commands/config.command.ts

```diff
@@ -10,6 +11,14 @@ export class ConfigCommand {
         }
     }
 
+    public async listVariables(jsonResponse: boolean, keysByVersion: string[], keysByVersionFile: string): Promise<void> {
+        if (jsonResponse) {
+            await variableService.exportVariables(keysByVersion, keysByVersionFile);
```

Comment:
I think this method name will be confusing when there is an actual export variables :smile:. Feel free to resolve and fix with variables export command.

Reply:
Hm, this is the PR for the variables export command. I think the confusion is that if jsonReponse is false you need to just list, otherwise you will export (as in export to a file)

Reply:
@LaberionAjvazi can you add a short example, of how the output is for both in the description?

Reply:
We discussed with @promeris and @jetakasabaqi regarding the naming of commands and wanted to switch to using `list variables` instead of `listVariables`. However, this was not possible with the current setup as the commander could not parse this command format. We will try to tackle this issue together with other import/export commands before releasing.